kedro/extras/extensions/ipython.py

```diff
@@ -58,7 +58,6 @@ def reload_kedro(
 
     _remove_cached_modules(metadata.package_name)
 
-    configure_project(metadata.package_name)
```

Comment:
The `bootstrap_project` method already included the `configure_project`, so I don't think it is needed

Reply:
I've been meaning to do this for ages. I didn't do it before because at a glance I wasn't 100% sure whether it was intentional to have both `bootstrap_project` and `configure_project` here. Note there's a call to `_remote_cached_modules` in between - does that mean that actually we need to repeat the call to `configure_project`?

I'm very much in favour of removing this if we can, but please do double check to see that all still works correctly after removing it! Possibly there's a good reason it's there.

Reply:
Sounds like you are confident we can remove this, so please go ahead as you were before 👍